recipes/openassetio/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "1.0.0-alpha.6":
```

Comment:
[fix] Should this be:
```suggestion
  "1.0.0-alpha.7":
```

Reply:
Oh dear, that's worrying, definitely updated that... checking.

Reply:
Nope genuinely missed that one, good spot, fixed in f92d01e8